### ISSUE TITLE
DynamicEntry does not inherit the environment_id from the ContentType

### ISSUE THREAD

**Issue by ahmad-bakr** [contentful/contentful-management.rb#176]
Given two environments (master and staging). 

```
client = Contentful::Management::Client.new(<token>)
content_type = client.content_types(<space_id>, 'staging').find(<content_type_id>)

p content_type.environment_id 
 => 'staging'
p content_type.entries.new.environment_id 
 => 'master'
``` 

That allows entries created this way to be stored in a different environment. I believe that is due environment_id is not passed to the DynamicEntry https://github.com/contentful/contentful-management.rb/blob/master/lib/contentful/management/content_type_entry_methods_factory.rb#L42

**Comment by dlitvakb** [comment]
Hey @ahmad-bakr,

This happens because as the entry doesn't have a `sys['environment']` it assumes it's defaulting to `master`.

This does not affect the behaviour of the entry. If you found a case, other than the snippet shared above, in which this affects your application, please let me know.

Cheers

**Comment by dlitvakb** [comment]
This has been fixed by #195 

Will release soon.

**Comment by dlitvakb** [comment]
Released in 2.10.0
